src/components/rangeInput.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from "react"
+
+import Input from "./input/index"
+
+import Label from "./fieldHelpers/label"
+
+interface Props {
+  name: {
+    min: string
+    max: string
+  }
+  handleChangeMin: (value) => void
+  handleChangeMax: (value) => void
+  value?: {
+    min: number
+    max: number
+  }
+  isValid?: {
+    min: boolean
+    max: boolean
+  }
+  placeholder?: {
+    min: string
+    max: string
+  }
+  required?: boolean
+  rangeInputLabel?: string
+  initialValue?: string
+}
+
+const RangeInput: FC<Props> = ({
+  name: { min: minName, max: maxName },
+  handleChangeMin,
+  handleChangeMax,
+  value: { min: minValue = "", max: maxValue = "" },
+  placeholder: { min: minPlaceholder = "", max: maxPlaceholder = "" },
+  rangeInputLabel,
+  required,
+}) => {
+  return (
+    <>
+      {rangeInputLabel ? (
+        <Label
+          label={rangeInputLabel}
+          htmlFor={`${minName}-min`}
+          required={required}
+        />
+      ) : null}
+      <div className="flex flex-wrap">
+        <div className="w-6/12">
+          <Input
+            name={minName}
```

Comment:
The input field already supports label. Can you please check if you could leverage it here too?
Something like:
```jsx
<Input
            labelText={rangeInputLabel ? rangeInputLabel : null}
            required={required}
```
on the first input field.
Then you may drop the new label component

Reply:
I tried your suggestion and it does not work because we have 2 inputs and they are not on the same level anymore. I guess I could make it work with some css, but it sounds a bit shaky